File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.Repository;

namespace SetPopupSize {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void gridLookUpEdit1_QueryPopUp(object sender, CancelEventArgs e) {
            GridLookUpEdit editor = (GridLookUpEdit)sender;
            RepositoryItemGridLookUpEdit properties = editor.Properties;
            properties.PopupFormSize = new Size(editor.Width - 4, properties.PopupFormSize.Height);
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.Repository

Namespace SetPopupSize
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub gridLookUpEdit1_QueryPopUp(ByVal sender As Object, ByVal e As CancelEventArgs) Handles gridLookUpEdit1.QueryPopUp
			Dim editor As GridLookUpEdit = CType(sender, GridLookUpEdit)
			Dim properties As RepositoryItemGridLookUpEdit = editor.Properties
			properties.PopupFormSize = New Size(editor.Width - 4, properties.PopupFormSize.Height)
		End Sub
	End Class
End Namespace
